primitives/core/storage/src/lib.rs

```diff
@@ -156,3 +159,155 @@ impl<'a> ChildStorageKey<'a> {
 		self.storage_key.into_owned()
 	}
 }
+
+#[derive(Clone, Copy)]
+/// Information related to a child state.
+pub enum ChildInfo<'a> {
+	Default(ChildTrie<'a>),
+}
+
+/// Owned version of `ChildInfo`.
+/// To be use in persistence layers.
+#[derive(Debug, Clone)]
+#[cfg_attr(feature = "std", derive(PartialEq, Eq, Hash, PartialOrd, Ord))]
+pub enum OwnedChildInfo {
```

Comment:
I feel like usually it goes the other way around with ChildInfo and ChildInfoRef.
But I'm completely fine with that